foxtrot-core/src/test/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchQueryStoreTest.java

```diff
@@ -393,14 +390,16 @@ public void testGetFieldMappings() throws FoxtrotException, InterruptedException
         Thread.sleep(500);
 
         Set<FieldTypeMapping> mappings = new HashSet<FieldTypeMapping>();
-        mappings.add(new FieldTypeMapping("word", FieldType.STRING));
-        mappings.add(new FieldTypeMapping("data.data", FieldType.STRING));
-        mappings.add(new FieldTypeMapping("header.hello", FieldType.STRING));
+        mappings.add(new FieldTypeMapping("word", FieldType.KEYWORD));
+        mappings.add(new FieldTypeMapping("data.data", FieldType.KEYWORD));
+        mappings.add(new FieldTypeMapping("header.hello", FieldType.KEYWORD));
         mappings.add(new FieldTypeMapping("head.hello", FieldType.LONG));
+        mappings.add(new FieldTypeMapping("timestamp", FieldType.DATE));
 
         TableFieldMapping tableFieldMapping = new TableFieldMapping(TestUtils.TEST_TABLE_NAME, mappings);
         TableFieldMapping responseMapping = queryStore.getFieldMappings(TestUtils.TEST_TABLE_NAME);
 
+        System.out.println(responseMapping.toString());
```

Comment:
Remove please.

Reply:
Not mine. Didn't see this. Will remove in next PR.